packages/core/src/sdk/deliveryPromise/useDeliveryPromise.ts

```diff
@@ -32,6 +32,12 @@ export const DYNAMIC_ESTIMATE_FACET_KEY = 'dynamic-estimate' as const
 
 type Facet = SearchState['selectedFacets'][number]
 
+type Tag = {
```

Comment:
question: could we import it from '@generated/graphql'?

Reply:
niceee. Updated [here](https://github.com/vtex/faststore/pull/2979/commits/ad7a9df267819cfb66a0c73ef8ecf767a58dc5be)